01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,64 @@
+const { writeFile, readFile } = require("fs").promises;
+
+async function writer() {
+  try {
```

Comment:
Instead of one big `try-catch` block, there should be one for each `writeFile(...)` statement.

Reply:
Let me know if you would like a little help. I may be able to meet with you